extensions/rolling/filter_test.go

```diff
@@ -178,6 +178,64 @@ func TestFiltersManagement(t *testing.T) {
 	})
 }
 
+func Disabled_BenchmarkOperationsParallel(b *testing.B) {
```

Comment:
don't use underscores in Go names; func Disabled_BenchmarkOperationsParallel should be DisabledBenchmarkOperationsParallel